packages/itwin/grouping-mapping-widget/src/widget/utils.ts

```diff
@@ -5,3 +5,11 @@
 export type CreateTypeFromInterface<Interface> = {
   [Property in keyof Interface]: Interface[Property];
 };
+
+export function debounce<F extends (...args: any[]) => void>(f: F, delay: number) {
+  let timer: NodeJS.Timeout;
```

Comment:
Lets steer away from using Node.js types, this will cause issues in the browser due to the Node API not being available. Eventually, we are switching to Webpack 5, which gets rid of the automatic pollyfilling of the Node.js APIs. Try using the window object.